meta-facebook/at-cb/src/platform/plat_isr.c

```diff
@@ -462,6 +462,7 @@ void ISR_POWER_STATUS_CHANGE()
 {
 	get_acb_power_status();
 	if (get_acb_power_good_flag()) {
+		init_clk_gen_spread_spectrum_control_register();
```

Comment:
The return value of this function doesn't seem to be used, should we either do some check or make the function return type as `void`?

Reply:
Hi @wangx6f 
As per the recommendation, I have modified this function to return void.